auth/api/iam/localize.go

```diff
@@ -0,0 +1,28 @@
+package iam
+
+import (
+	"strings"
+)
+
+// localize selects from a range of localized values based on the accept-language header, returning
+// a default value if no localized value matches the languages specified in the accept-language header.
+func localize(defaultValue string, localizedValues map[string]string, acceptLanguageHeader string) string {
```

Comment:
I guess browsers always send their preferred language?

Reply:
Never mix the words "browser" and "always" in a sentence. We will just fall back to default if it is blank, which I should create a unit test for.